pkg/fdnotifier/BUILD

```diff
@@ -10,5 +10,7 @@ go_library(
     ],
     importpath = "gvisor.dev/gvisor/pkg/fdnotifier",
     visibility = ["//:sandbox"],
-    deps = ["//pkg/waiter"],
+    deps = ["//pkg/waiter",
```

Comment:
Please put waiter on its line (that's the standard style).

Note that [buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier) can auto-format BUILD files.

Reply:
Fixed! Will try to enable the buildfier tool in our environment. Thansk!